readme.md

```diff
@@ -0,0 +1,54 @@
+# Abstract
+Our idea is to implement a robot which can perform SLAM (Simultaneous Localization and Mapping) autonomously in any surrounding. This automated bot can be deployed in various situations which are risky for humans such as in remote survelliance, etc. 
+
+# Pre-requisite and Software Used
+This whole work is being developed in ROS(Robot Operating System). It is being chosen due to its simple usage to implement various different components and pacjkages used for robot  building and to integrate them easily. To get basic ROS usage guidelines, refer this link: [ROS Documentaion](http://wiki.ros.org/) </br>
```

Comment:
Please remove the <br/> tags. Paragraphs in markdown are denoted by a black line.